godotproject/Player.tscn

```diff
@@ -281,11 +281,8 @@ blend_point_3/node = SubResource( 23 )
 blend_point_3/pos = Vector2( 0, -1 )
 blend_mode = 1
 
-[sub_resource type="AnimationNodeAnimation" id=83]
-animation = "Hurt"
-
 [sub_resource type="AnimationNodeAnimation" id=25]
-animation = "Idle_froggie"
+animation = "Hurt"
```

Comment:
Not asking for any change to the PR here, but just.. wtf is going on in this file??? Godot decided to just randomly change the IDs of all the animations or smth?

Reply:
yeah idk, anytime I update a .gd file, godot always does some weird stuff with the tscn file. It still runs fine so I generally just ignore it